src/amo/components/Header/index.js

```diff
@@ -69,6 +69,14 @@ export class HeaderBase extends React.Component {
         <SectionLinks className="Header-SectionLinks" location={location} />
 
         <div className="Header-user-and-external-links">
+          <Link
+            className="Header-extension-workshop-link Header-button"
+            href="https://extensionworkshop.com/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=header-link"
```

Comment:
@William I need some help here, right now the link which is rendering 

`localhost:3000/en-US/https:/extensionworkshop.com/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=header-link` which is weird. `https://` is converting into `https:/`. Can you please help me here whenever you have time.

Thanks

Reply:
mmh, I suppose we should add `prependLang={false}` and `prependClientApp={false}`

Reply:
@willdurand Thanks for the help made the changes ready for another review :D